.travis.yml

```diff
@@ -23,11 +23,7 @@ dist: trusty
 
 addons:
   firefox: latest
-  apt:
-    sources:
-      - google-chrome
-    packages:
-      - google-chrome-stable
+  chrome: beta # stable version (60.0.3112.78) is bugged
```

Comment:
What's the bug exactly? are we sure it's not our code that doesn't work against this version?